docs/dev/python/user-guide/table/10_minutes_to_table_api.md

```diff
@@ -0,0 +1,712 @@
+table_env.execute_sql("""
+    CREATE TABLE sql_source (
+        id BIGINT, 
+        data TINYINT) 
+    WITH (
+        'connector' = 'datagen',
+        'fields.id.kind'='sequence',
+        'fields.id.start'='1',
+        'fields.id.end'='4',
+        'fields.data.kind'='sequence',
+        'fields.data.start'='4',
+        'fields.data.end'='7'
+    )
+""")
+
+# convert the sql table to Table API table
+table = table_env.from_path("sql_source")
+
+# or create the table from a sql query
+table = table_env.sql_query("SELECT * FROM sql_source")
+
+# emit the table
+table.to_pandas()
+
+{% endhighlight %}
+
+{% top %}
+
+Emit Results
+------------
+
+### Emit to Variable
+
+You can call "to_pandas" method to emit the data in a `Table` object to a pandas DataFrame:
+
+{% highlight python %}
+
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table.to_pandas()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+   id   data
+0   1     Hi
+1   2  Hello
+{% endhighlight %}
+
+Note that "to_pandas" is not supported on flink planner, and not all data types can be emitted to pandas DataFrame.
+If the `TableEnvironment` is in streaming mode, calling "to_pandas" on the tables which contain retract messages (i.e. update-before) is also not supported.
+
+### Emit to Single Sink Table
+
+You can call "execute_insert" method to emit the data in a `Table` object to a sink table:
+
+{% highlight python %}
+
+table_env.execute_sql("""
+    CREATE TABLE sink_table (
+        id BIGINT, 
+        data VARCHAR) 
+    WITH (
+        'connector' = 'print'
+    )
+""")
+
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table.execute_insert("sink_table").get_job_client().get_job_execution_result().result()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+6> +I(1,Hi)
+6> +I(2,Hello)
+{% endhighlight %}
+
+The equivalent SQL way is:
+
+{% highlight python %}
+
+table_env.create_temporary_view("table_source", table)
+table_env.execute_sql("INSERT INTO sink_table SELECT * FROM table_source") \
+    .get_job_client().get_job_execution_result().result()
+
+{% endhighlight %}
+
+### Emit to Multiple Sink Tables
+
+You can use the `StatementSet` to emit the `Table`s to multiple sink tables in one job:
+
+{% highlight python %}
+
+# prepare source tables and sink tables
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table_env.create_temporary_view("simple_source", table)
+table_env.execute_sql("""
+    CREATE TABLE first_sink_table (
+        id BIGINT, 
+        data VARCHAR) 
+    WITH (
+        'connector' = 'print'
+    )
+""")
+table_env.execute_sql("""
+    CREATE TABLE second_sink_table (
+        id BIGINT, 
+        data VARCHAR) 
+    WITH (
+        'connector' = 'print'
+    )
+""")
+
+# create a statement set
+statement_set = table_env.create_statement_set()
+
+# accept a Table API table
+statement_set.add_insert("first_sink_table", table)
```

Comment:
As someone who doesn't know the API very well, I don't understand what's happening here. Also, "accept" doesn't feel like the correct verb, but as I don't fully understand what this code is doing, I'm not sure what to suggest instead.

Reply:
I'll replace these comments with `# emit the "table" object to the "first_sink_table"` and `# emit the "simple_source" to the "second_sink_table" via a insert sql query`. Do these seem more readable?